raster-foundry/raster-foundry
# Upgrade STAC version for exports

## Overview

This PR：
- removes the dependency on `geotrellis-server-stac`
- replaces it with `stac4s` so that the version of the STAC exports comply to `1.0.0-beta.1` of STAC spec
- removes some STAC label item related data models that are irrelevant after having `stac4s` 
- fixed the "key" of the label assets to be `"labels"`
- updates the version of `refined` to make tests happier.

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~
- ~Any new endpoints have scope validation and are included in the integration test csv~

## Testing Instructions

- It should all compile just fine
- I pushed a `test` branch to staging, so that the new STAC export can be tested on staging from any of your Groundwork project
- So, log in to GW staging, go to one of your own projects, kick off an export
- The export should succeed. When it become available, download it and make sure the catalog, the collections, the label item and the image items are right and compliant with the version  `1.0.0-beta.1` of STAC spec

Closes https://github.com/raster-foundry/raster-foundry/issues/5299
